Source/SwiftLintBuiltInRules/Rules/Lint/UnneededOverrideRuleExamples.swift

```diff
@@ -137,6 +137,13 @@ struct UnneededOverrideRuleExamples {
             }
         }
         """),
+        Example("""
+        class Foo {
```

Comment:
For the sake of completeness:
```suggestion
        class Foo: XCTestCase {
```